README.md

```diff
@@ -71,11 +71,11 @@ like.
 ### Local preview
 
 1. Remove `public` directory: `rm -rf blog/public`
-1. Generate blog: `docker run --rm -it -v $PWD/blog:/src -u hugo jguyomard/hugo-builder:0.54 hugo`
+1. Generate blog: `sudo docker run --rm -it -v "$PWD/blog:/src" -u hugo jguyomard/hugo-builder:0.54 hugo`
```

Comment:
@dawidzebacki3mdeb there is no need to do that. Please check this:
https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user

I will revert these changes

Reply:
@artur-rs I know that there is no need to do that but as the first time user, I encountered an error associated with this part of code. I changed this so that it works for everyone, not only non-root users. If you think that there is no need to change README.md that's fine.